packages/headless/src/features/case-context/case-context-state.ts

```diff
@@ -0,0 +1,7 @@
+export interface CaseContextState {
+  caseContext: Record<string, string>;
+}
+
+export const getCaseContexttInitialState = (): CaseContextState => ({
```

Comment:
```suggestion
export const getCaseContextInitialState = (): CaseContextState => ({
```

Also the function should be documented.